README.md

```diff
@@ -7,15 +7,15 @@ For more information about Keycloak visit the [Keycloak homepage](http://keycloa
 
 
 Contributing to Keycloak Documentation
---------------------------------------
+----------------------------------------
```

Comment:
Just a nitpick (not a blocker). Why to make this change? Seems unrelated (and even in an unrelated file).